csaf_2.1/prose/edit/src/design-considerations-01-construction-principles.md

```diff
@@ -34,8 +34,17 @@ Proven and intended usage patterns from practice are given where possible.
 
 Delegation to industry best practices technologies is used in referencing schemas for:
 
+* Classification for Document Distribution
+  * Traffic Light Protocol (TLP)
+    * Default Definition: https://www.first.org/tlp/
+* Vulnerability Classification
```

Comment:
Can we move this down, so that we have some loose ordering and grouping like:

```
- Classification for Document Distribution
- Platform Data
- Vulnerability Categorization
- Vulnerability Classification
- Vulnerability Scoring
```

... and drop colons where only some same level entries have them or complete (my take is always to not have trailing punctuation when using these 2 dimensional lists engineers love soooo much :-)

Reply:
Done in [674e64d](https://github.com/oasis-tcs/csaf/pull/871/commits/674e64d4410482f6e12bf3d0b1140a14b05523f7)